app/pb_kits/playbook/pb_label_value/_label_value.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class)) do %>
-  <%= object.label %>
-  <%= object.value %>
+    class: object.classname) do %>
+  <%= pb_rails("caption", props: { text: object.label }) %>
+  <%= pb_rails("body", props: { text: object.value }) %>
```

Comment:
It doesn't look like the Body.new invocation was receiving the `text` prop. It looks like it was receiving the object's value via a block.

Reply:
We discussed this via a screen share and found out that this will work the same way for the body kit.